src/manage_pg.c

```diff
@@ -1606,41 +1605,14 @@ create_view_vulns ()
 {
   sql ("DROP VIEW IF EXISTS vulns;");
 
-  if (sql_int ("SELECT EXISTS (SELECT * FROM information_schema.tables"
```

Comment:
This block also handles the version of the `vulns` view that includes CVEs, so instead of removing it entirely you should just adjust the check to use the table `cves` and remove the last `UNION` clause that uses `ovaldefs`.